### PR TITLE
Changes all instances of `pick(list(....))` to `pick(....)`

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
Changes all instances of `pick(list(....))` to `pick(....)`, as it is ever so slightly faster and an example of a bad coding practice.
There's currently only one place where it is actually needed.

## Why It's Good For The Game
This is a standard we should follow, as it looks better and makes more sense to write it like this.

## Testing
Compiled, checked some of the random name functions for different species

## Changelog
NPFC

### CODE REVIEW

File: code/datums/diseases/kingstons.dm [hunk 1]
```diff
@@ -33,7 +33,7 @@
 					to_chat(affected_mob, "<span class='danger'>You feel something in your throat!</span>")
 					affected_mob.emote("cough")
 				else
-					affected_mob.say(pick(list("Mew", "Meow!", "Nya!~")))
+					affected_mob.say(pick("Mew", "Meow!", "Nya!~"))
```
Comment (GDNgit) [inline code/datums/diseases/kingstons.dm:36]: Nya!~

**Review by GDNgit — APPROVED** [review]

File: code/datums/diseases/kingstons.dm (hunk 1 above)
Comment (DGamerL) [reply code/datums/diseases/kingstons.dm:36]: Nyaaa :3

**Review by Henri215 — APPROVED** [review]
Nyaaaaaa

**Comment by DGamerL** [comment]
I'll be removing the last place probably aswell, please hold off on merging until tonight

**Review by Burzah — APPROVED** [review]

**Comment by DGamerL** [comment]
> I'll be removing the last place probably aswell, please hold off on merging until tonight

Ready for merge, the list is there for now so it's actually readable

**Review by SteelSlayer — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

File: .github/CONTRIBUTING.md
Comment (farie82) [inline .github/CONTRIBUTING.md:1]: Can you add the concrete guideline here as well?

**Review by farie82 — APPROVED** [review]

**Pull request merged**